scripts/publish.ts

```diff
@@ -6,8 +6,12 @@ import execa from 'execa'
 import slugify from '@sindresorhus/slugify'
 import arg from 'arg'
 
-function branchToTag(branch: string): string {
-  if (branch === 'master') {
+/**
+ * Turns a prisma-engines Git branch name (that got new engines published) into a 
+ * engines-wrapper Npm dist tag name (where the local packages will be published)
+ */
+function prismaEnginesBranchToNpmDistTag(branch: string): string {
+  if (branch === 'master' || branch === 'main') {
```

Comment:
This is prep for when we rename the main branch to be less racist.

Reply:
Noice